LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-changes-overflow-left.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<meta name="timeout" content="long">
+<html class=reftest-wait>
+<title>View transitions: capture elements and then change overflow</title>
+<link rel="help" href="https://github.com/WICG/view-transitions">
```

Comment:
I would use this link: 
```suggestion
<link rel="help" href="https://drafts.csswg.org/css-view-transitions-1/">
```